docs/StardustDocs/topics/flatten.md

```diff
@@ -11,13 +11,36 @@ flatten  [ { columns } ]
 Columns after flattening will keep their original names. Potential column name clashes are resolved by adding minimal possible name prefix from ancestor columns.
 
 <!---FUN flatten-->
```

Comment:
you renamed this function to `flatten_properties` in Modify.kt

We use Korro to check that samples in stardust are actually compilable. So make sure that all examples in markdown are surrounded like
`<!---FUN someFunctionName-->` 
your example, which is between // SampleStart and // SampleEnd comments in samples/api/Modify.kt inside a function with the same name.
`<!---END-->`

Reply:
Template looks like: name of function underscore acess api

Reply:
Oh now I see! Didn't know this was possible. Cool :)